HarpGuru/HarpFace/HarpCell/components/index.tsx

```diff
@@ -14,16 +15,23 @@ import { setPozitionRootAtCell } from '../setPozitionRootAtCell'
 import { getDisplayValue } from '../getDisplayValue'
 import { analysePosition } from '../analysePosition'
 
-export const HarpCell = (props: HarpCellProps): React.ReactElement => {
-  const { setActiveHarpStrata, activeHarpStrata } = props
-  const positionFacts = analysePosition(props)
+export const HarpCell = (
+  props: Pick<HarpCellProps, 'yxCoord' | 'activeDisplayMode'>
```

Comment:
Obviously it's a bit odd that the HarpCell itself is no longer using the full list of HarpCellProps. It's actually inconsistent with how we've made changes elsewhere too where the main component props have actually had the HarpStrata props removed from them and all the dependencies which formally inherited the HarpStrata props in to their props just had to manually add them in now. We should modify the way this one works to be consistent with the rest of the change.

Reply:
On closer inspection, all that had happened was that I had forgotten to take the `Pick` off of these props. The HarpCellProps have already had the HarpStrata props removed from them so this `Pick` was actually doing nothing.

Fixed.